test/test_dmmot.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for ot.lp.dmmot module """
+
+# Author: Ronak Mehta <ronakrm@cs.wisc.edu>
+#         Xizheng Yu <xyu354@wisc.edu>
+#
+# License: MIT License
+
+import numpy as np
+import ot
+
+
+def create_test_data():
+    np.random.seed(1234)
+    n = 4
+    a1 = ot.datasets.make_1D_gauss(n, m=20, s=5)
+    a2 = ot.datasets.make_1D_gauss(n, m=60, s=8)
+    A = np.vstack([a1, a2])
+    x = np.arange(n, dtype=np.float64).reshape((n, 1))
+    return A.T, x
+
+
+def test_discrete_mmot():
```

Comment:
since you implemented the function with backends, you should run the tests on arrays from other backends, you can do that by adding a parameter nx to the test function that will be automatically run with all available backends.